src/node_crypto.cc

```diff
@@ -1558,6 +1560,32 @@ void SecureContext::SetTicketKeys(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void SecureContext::GetNumTickets(const FunctionCallbackInfo<Value>& args) {
+  SecureContext* sc;
+  ASSIGN_OR_RETURN_UNWRAP(&sc, args.Holder());
+
+  CHECK_EQ(args.Length(), 0);
+
+  uint32_t numTickets = SSL_CTX_get_num_tickets(sc->ctx_.get());
+  args.GetReturnValue().Set(static_cast<uint32_t>(numTickets));
```

Comment:
```suggestion
  uint32_t num_tickets = SSL_CTX_get_num_tickets(sc->ctx_.get());
  args.GetReturnValue().Set(num_tickets);
```

(We use `lower_camel_case` for local variables. But it’s not really all that important anyway.)